mercury/views/can.py

```diff
@@ -0,0 +1,18 @@
+from django.http import HttpResponse
+from django.views.decorators.csrf import csrf_exempt
+import logging
+from ..can_decoder import decode_can_message
+
+logging.basicConfig(level=logging.DEBUG)
+log = logging.getLogger(__name__)
+
+
+@csrf_exempt
+def post(request, *args, **kwargs):
+    if not request.body:
+        """Return 400 Bad Request if no CAN message is received"""
+        return HttpResponse(status=400)
+    log.debug(request.body)
+    log.debug(dir(request.body))
+    decoded_message = decode_can_message(request.body)
+    return HttpResponse(decoded_message, status=201)
```

Comment:
For right now, I am just running the input through the `decode_can_message` function and returning it back to the user with a 201. It is not actually writing anything yet. My intention is to figure out which sensor to write a new data row of based on the ID in the can message, so that will happen in this `post` method.